app/controllers/general.php

```diff
@@ -157,6 +158,9 @@
     $responseFormat = $request->getHeader('x-appwrite-response-format', App::getEnv('_APP_SYSTEM_RESPONSE_FORMAT', ''));
     if ($responseFormat) {
         switch($responseFormat) {
+            case version_compare ($responseFormat , '0.12.3', '<='):
+                Response::setFilter(new FiltersV12());
```

Comment:
This naming is inconsistent with the filter we already have. 

Since we also have request and response filter, lets use `ResponseV12` and `ResponseV11` here and `RequestV12` for the request filters